pom.xml

```diff
@@ -264,6 +264,11 @@
       <id>run-its</id>
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.eclipse.sisu</groupId>
+            <artifactId>sisu-maven-plugin</artifactId>
+            <version>0.3.5</version>
+          </plugin>
```

Comment:
I think it is preferable to customise the compiler plugin and add the issue apt processor:
```
        <plugin>
          <groupId>org.apache.maven.plugins</groupId>
          <artifactId>maven-compiler-plugin</artifactId>
          <dependencies>
            <dependency>
              <groupId>org.eclipse.sisu</groupId>
              <artifactId>org.eclipse.sisu.inject</artifactId>
              <version>0.3.5</version>
            </dependency>
          </dependencies>
        </plugin>
```

Reply:
My point was that using the APT processor from the `org.eclipse.sisu.inject` artifact is faster than using the `sisu-maven-plugin`, so I'd rather remove the plugin config completely and use the processor instead.

Reply:
@gnodet  - In Maven parent Pom we use plugin ... maybe we should consider to use also processor ... one less plugin to execute

https://github.com/apache/maven-parent/blob/ba4c881d114135c38a5a0d7492238c3573ecac0b/pom.xml#L989-L1002